package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "movies-db",
   "version": "1.0.0",
-  "description": "react-redux movies database",
+  "description": "react movie database",
   "main": "index.js",
   "scripts": {
     "start": "webpack-dev-server",
```

Comment:
Consider adding "lint": "eslint src" command to run lint from console like so: "npm run lint"